src/components/pages/account/EarnWidget.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { DateTime, Interval } from 'luxon';
+
+import { useMainCurrency, useCashFlow } from '@/hooks/api';
+import type { Account } from '@/book/entities';
+import StatisticsWidget from '@/components/StatisticsWidget';
+import { moneyToString } from '@/helpers/number';
+
+export type EarnWidgetProps = {
+  account: Account,
+};
+
+export default function EarnWidget({
```

Comment:
This widget is almost a copy of SpendWidget except for some styling and filters. However separated because description field is a bit different and creates some complexity